src/main/webapp/dev/js/common/questionRubric.js

```diff
@@ -355,8 +355,34 @@ function highlightRubricCol(index, questionNum, highlight) {
 }
 
 /**
- * Shows the weight row if the "Choices are weighted" checkbox is checked, otherwise
- * hides the weight row.
+ * Makes all the weight cells required.
+ *
+ * @param $weightRow
+ */
+function makeRubricWeightsRequired($weightRow) {
```

Comment:
```javascript
function setRequiredAttributeToRubricWeight($weightRow, isRequired) {
    // this will short your code :)
}
```

Reply:
Of course! :man_facepalming: Why didn't I think about it earlier!